package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@youleap/types",
+  "version": "1.0.0",
+  "description": "@youleap/types for external and internal components, dashboards and functions.",
+  "license": "Apache License 2.0",
```

Comment:
Verify license
I think we need to add license file too if open source it

Reply:
we have a license file

Reply:
Where? I don't see it in the source code.
I write it because I once saw npm package inspect tool that alert about this rule

Reply:
There's a file called "[LICENSE](https://github.com/youleap/types/blob/main/LICENSE)", and changed the identifier to the correct one.